journal/202309.md

```diff
@@ -0,0 +1,971 @@
+
+
+<a id="bison-relay-v019-release" />
+
+## Bison Relay v0.1.9 Release
+
+{ highlights https://github.com/companyzero/bisonrelay/releases/tag/v0.1.9 }
+
+
+<a id="dcrdex-v063-release" />
+
+## DCRDEX v0.6.3 Release
+
+{ https://github.com/decred/dcrdex/releases/tag/v0.6.3 + better bins }
+
+
+<a id="development" />
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+
+### dcrd
+
+_[dcrd](https://github.com/decred/dcrd) is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+Changes backported to the v1.8.1 release:
+
+- Added the new [proof of work hash](https://github.com/decred/dcrd/pull/3192) to the verbose outputs of `getblock` and `getblockheader`.
+- Changed the [`getnetworkhashps`](https://github.com/decred/dcrd/pull/3181) to treat `-1` as a default value (120 blocks back). Previous meaning of `-1` "the number of blocks since last difficulty change" no longer makes sence since the difficulty is now calculated every block as opposed to every 144 blocks with the old difficulty algorithm.
+
+Changes merged in `master` towards future releases:
+
+- Updated Docker image to [Go 1.21.1](https://github.com/decred/dcrd/pull/3183).
+- Improved internal [error handling](https://github.com/decred/dcrd/pull/3182) in the `rpcserver` to make the code cleaner and harder to misuse.
+- Changed [`sampleconfig`](https://github.com/decred/dcrd/pull/3185) package to use Go's embed functionality. It allows to have sample configs as separate files that are easier to work with, while retaining convenient access to their contents from code.
+- Added functions to access the [R and S values](https://github.com/decred/dcrd/pull/3188) of signatures in the `secp256k1/ecdsa` package, these values can be useful in some computations.
+- Added a couple of [new](https://github.com/decred/dcrd/pull/3190) [tests](https://github.com/decred/dcrd/pull/3191) to push test coverage to 99.6%.
+- Minor changes { _drop after review: 3186, 3189_ }
+
+
+### dcrwallet
+
+_[dcrwallet](https://github.com/decred/dcrwallet) is a wallet server used by command-line and graphical wallet apps._
+
+Changes backported to the v1.8.1 release:
+
+- Updated verbose [`getblock` and `getblockhash`](https://github.com/decred/dcrwallet/pull/2285) responses to include the new proof of work hash.
+
+Changes merged in `master` towards future releases:
+
+- Updated [VSP fee calculation](https://github.com/decred/dcrwallet/pull/2281) for the new block reward split in DCP-12 (proof of stake reward increased from 80% to 89%).
+  - { _if this is not backported v1.8.1, will VSPs accept unupdated fees?_ }
```

Comment:
The updated piece of code is only used by vspd itself, not by vspd clients (ie ticket buyers), so theres no need for it to be backported.

Reply:
Updated.